src/scripting_api/util.js

```diff
@@ -536,15 +540,15 @@ class Util extends PDFObject {
 
     const [re, actions] = this._scandCache.get(cFormat);
 
-    const matches = new RegExp(re, "g").exec(cDate);
+    const matches = new RegExp(`^${re}$`, "g").exec(cDate);
     if (!matches || matches.length !== actions.length + 1) {
       return null;
     }
 
     const data = {
-      year: 0,
+      year: 2000,
       month: 0,
-      day: 0,
+      day: 1,
```

Comment:
Are these default values changed to match the Adobe JS specification, or simply to match the implementation in Adobe Reader?

Reply:
I opened the pdf from the issue in Acrobat and the UI for the date selection is set to January 2000 for the day field and to the 1st of December 2000 for the month field.